gunicorn/http/wsgi.py

```diff
@@ -300,10 +299,11 @@ def default_headers(self):
         headers = [
             "HTTP/%s.%s %s\r\n" % (self.req.version[0],
                 self.req.version[1], self.status),
-            "Server: %s\r\n" % self.version,
             "Date: %s\r\n" % util.http_date(),
             "Connection: %s\r\n" % connection
         ]
+        if self.cfg and self.cfg.server_tokens:
```

Comment:
I haven't tested this, but with reference to my previous comments - if you can set an empty string then this is falsey and will be skipped.  Do we still need true/True/false/False?